.gitignore

```diff
@@ -159,3 +159,5 @@ dask-worker-space/
 
 # protobuf
 **/*_pb2.py
+
+rmm_log.txt
```

Comment:
Can the changes to this file be removed?

Reply:
sure, done